mongoDB_ODS/app/json_encoder.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Digital Asset (Switzerland) GmbH and/or its affiliates.
```

Comment:
How do I use this? Can I get more info please? A quick Google search didn't turn me up anything immediately usable ...

Reply:
`dade` stands for "digital asset developer environment". It's an internal tool so google is no use.
I think you just run the script inside a `git` repo and it adds headers to all files. Otherwise ask Brian.

Reply:
Addressed this